### ISSUE TITLE
No console displaying

### ISSUE THREAD

**Issue by olivierlambert** [HPieters/xenapi-jquery#1]
Hi, 

Test page is OK, but VNC is not: I'm stuck with "Starting VNC handshake". Sometimes, when I manually load the console URL in a tab and refresh VNC page, it works, but not every time. 

Maybe I miss something, a step in order to unlock the console, I don't know.

**Comment by olivierlambert** [comment]
Debug on XenServer log:
xapi: [debug|lab2|359 INET 0.0.0.0:80|Connection to VM console R:f34ceedee862|console] VM OpaqueRef:aff0fe26-0ace-5e5d-f8b6-37322c7ebdc7 console port: Some 9501

But nothing is displayed. After multiple page refresh, it works (sometimes, approx. once in 20 trials). Tested on fresh install of XenServer 6.2, with no modifications.

**Comment by HPieters** [comment]
Yes the VNC example sadly has some context specific dependancies based on my own setup. I fixed this in my latest push. However I am getting some weird behavior of my own after investigating your reports, I will continue to investigate this further...
